src/cmds/mem/infomem.c

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file
+ * @brief
+ *
+ * @author  Evgenij Bogdanov
+ * @date    09.07.2018
+ */
+
+#include <stdio.h>
+
+#include <mem/phymem.h>
+#include <mem/vmem/vmem_alloc.h>
+
+#include <mem/vmem.h>
+#include <drivers/common/memory.h>
+
+static void print_vmem(struct page_allocator *pgd_allocator,
+		struct page_allocator *pmd_allocator, struct page_allocator *pte_allocator) {
+	int is_on_vmem = vmem_mmu_enabled();
+
+	printf("VIRTUAL MEMORY:\n");
+	printf("IS_ON - %d\n", is_on_vmem);
```

Comment:
I recommend to use something like `printf("Virtual memory is %s\n", is_on_vmem ? "on" : "off");`, it's more clear.

Reply:
Fixed